src/PublicKey.cpp

```diff
@@ -86,18 +83,16 @@ PublicKey PublicKey::compressed() const {
     }
 
     Data newBytes(secp256k1Size);
+    assert(bytes.size() >= 65);
     newBytes[0] = 0x02 | (bytes[64] & 0x01);
 
-    switch (type) {
-    case TWPublicKeyTypeSECP256k1Extended:
```

Comment:
can just do `if, else if, else`? the default case is removed, less readable for me

Reply:
OK, made back to 2-branch-switch, more readable.